### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by nsaquino**
Update the game over message so people know how to play again!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,51 +14,45 @@ _Collaborate and work together on GitHub._
 </header>
 
 <!--
-  <<< Author notes: Course start >>>
-  Include start button, a note about Actions minutes,
-  and tell the learner why they should take the course.
+  <<< Author notes: Step 3 >>>
+  Start this step by acknowledging the previous step.
```
Comment (nicolas-aquino) [inline README.md:18]:
```suggestion
  Start this step by acknowledging the previous step..
```

**Review by nicolas-aquino — CHANGES_REQUESTED** [review]
test

**Review by nicolas-aquino — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ _Collaborate and work together on GitHub._
 
 <!--
   <<< Author notes: Step 4 >>>
-  Start this step by acknowledging the previous step.
+  Start this step by acknowledging the previous step..
```
Comment (nicolas-aquino) [inline README.md]:
```suggestion
  Start this step by acknowledging the previous step.
```

**Review by nicolas-aquino — CHANGES_REQUESTED** [review]
test

**Review by GrumblyProcess — DISMISSED** [review]
bien

**Review by nicolas-aquino — APPROVED** [review]

**Pull request merged**